ika/tests/test_shaker_driver.py

```diff
@@ -71,23 +63,37 @@ async def test_setpoint_roundtrip():
     """Confirm that the setpoint can be updated."""
     async def get():
         async with Shaker(ADDRESS) as device:
-            with pytest.raises(ValueError):
```

Comment:
Don't remove this test!

https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions